zeppelin-plugins/launcher/docker/src/main/java/org/apache/zeppelin/interpreter/launcher/DockerInterpreterLauncher.java

```diff
@@ -54,11 +57,11 @@ public InterpreterClient launchDirectly(InterpreterLaunchContext context) throws
 
     StandardInterpreterLauncher interpreterLauncher = null;
     if (isSpark()) {
-      interpreterLauncher = new SparkInterpreterLauncher(zConf, recoveryStorage);
+      interpreterLauncher = (StandardInterpreterLauncher) pluginManager.createInterpreterLauncher("SparkInterpreterLauncher", recoveryStorage);
     } else if (isFlink()) {
-      interpreterLauncher = new FlinkInterpreterLauncher(zConf, recoveryStorage);
+      interpreterLauncher = (StandardInterpreterLauncher) pluginManager.createInterpreterLauncher("FlinkInterpreterLauncher", recoveryStorage);
```

Comment:
FlinkInterpreterLauncher ？

Reply:
I think it makes no difference, but I changed it.